### DISCUSSION TITLE
Easy mode flags for custom mode

### DISCUSSION THREAD

**Discussion by shaolo1** [complexlogic/rsgain#82] [General]
I need to run gain calculations on a read only file system, so I obviously won't be writing tags to the tracks.  As for the rest of the easy mode defaults is the following the custom mode equivalent?
rsgain custom --loudness=-18 --clip-mode=p --max-peak=0 track.flac

**Comment by complexlogic** [comment]
Yes, that will work.

**Comment by shaolo1** [comment]
It seemed right from the documentation, but I wasn't 100% sure.  Thanks!
